### PR TITLE
Extract submodule

### PR DESCRIPTION
**Pull request by Ndpnt**

### CODE REVIEW

**Comment by Ndpnt** [comment]
Data path should be in config. Wait for #26 to be merged.

File: ops/roles/cgus-data/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+- name: Add github ssh key
```
Comment (MattiSG) [inline ops/roles/cgus-data/tasks/main.yml:1]:
```suggestion
- name: Add GitHub SSH key
```

File: ops/roles/cgus-data/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  copy:
+    src: cgus-data.private_key
+    dest: "/home/{{ ansible_user }}/.ssh/cgus-data"
+    owner: "{{ ansible_user }}"
+    group: "{{ ansible_user }}"
+    mode: 0600
+  tags:
+    - setup
+
+- name: Configure ssh to use ansible key for github.com
```
Comment (MattiSG) [inline ops/roles/cgus-data/tasks/main.yml:11]:
```suggestion
- name: Configure SSH to use Ansible key for github.com
```

File: ops/roles/cgus-data/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  template:
+    src: ssh_config.j2
+    dest: "/home/{{ ansible_user }}/.ssh/config"
+    owner: "{{ ansible_user }}"
+    group: "{{ ansible_user }}"
+    mode: 0644
+  tags:
+    - setup
+
+- name: Clone Repo
```
Comment (MattiSG) [inline ops/roles/cgus-data/tasks/main.yml:21]:
```suggestion
- name: Clone repository
```

File: src/history/git.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ import simpleGit from 'simple-git';
 
 const __dirname = path.dirname(new URL(import.meta.url).pathname);
 
-const DATA_PATH = '../../data';
+const DATA_PATH = process.env.CI ? '../../CGUs-data': '../../../CGUs-data';
```
Comment (MattiSG) [inline src/history/git.js:9]:
```suggestion
const DATA_PATH = (process.env.CI ? '../' : '') + ../../CGUs-data';
```

Comment (MattiSG) [inline src/history/git.js:9]: > I don't really understand why we need this, and I'm really not a fan of having production code that does disjunctions based on CI / tests environments 😕

File: src/history/persistor.js [hunk 5]
```diff
@@ -14,7 +14,7 @@ import { DOCUMENTS_TYPES } from '../documents_types.js';
 
 const __dirname = path.dirname(new URL(import.meta.url).pathname);
 
-const ROOT_DIRECTORY = path.resolve(__dirname, `../../data`);
+const ROOT_DIRECTORY = path.resolve(__dirname, process.env.CI ? '../../CGUs-data': '../../../CGUs-data');
```
Comment (MattiSG) [inline src/history/persistor.js:17]:
```suggestion
const ROOT_DIRECTORY = path.resolve(__dirname, process.env.CI ? '..' : '', '../../CGUs-data');
```

File: src/history/git.js (hunk 4 above)
Comment (Ndpnt) [reply src/history/git.js:9]: Yep, I undestand. I'm already fixing this in another branch with the config module

**Pull request merged**